### PR TITLE
Add functions from content

### PR DESCRIPTION
**Pull request by liorblob**
## Status
Ready

## Related Issues
https://github.com/demisto/etc/issues/22938

## Description
As part of the content repo cleanup, we're moving all the constants and tools to the SDK so there will be no inconsistencies.

### CODE REVIEW

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -648,3 +648,384 @@ def run_command_os(command: str, cwd: Path, env: dict = os.environ) -> Tuple[str
...
+    :return: tag
+    """
+    tags = run_command('git tag').split('\n')
+    tags = [tag for tag in tags if re.match(r'\d+\.\d+\.\d+', tag) is not None]
+    tags.sort(key=LooseVersion, reverse=True)
+
+    return tags[0]
+
+
+def collect_pack_content_items(pack_path):
```
Comment (IkaDemisto) [inline demisto_sdk/commands/common/tools.py]: You can remove it, will be moved to store_packs

File: demisto_sdk/commands/common/tools.py [hunk 2]
```diff
@@ -648,3 +648,384 @@ def run_command_os(command: str, cwd: Path, env: dict = os.environ) -> Tuple[str
...
+
+            data[directory] = dir_data
+    except Exception as e:
+        task_status = False
+        print_error("Failed to collect pack content items at path :{}\n. Additional info {}".format(pack_path, e))
+    finally:
+        return task_status, data
+
+
+def input_to_list(input_data):
```
Comment (IkaDemisto) [inline demisto_sdk/commands/common/tools.py]: Same here

**Comment by liorblob** [comment]
@yuvalbenshalom The only addition here is the `Docker` class, which is difficult to test because it uses SSH. We can choose to skip testing that for now or have a zoom about it with @IkaDemisto .

**Comment by IkaDemisto** [comment]
@liorblob do we need the `Docker` class in demisto-sdk? In is for our internal use, mainly in `test_integration`. Are we planning to move it also to sdk?

**Comment by liorblob** [comment]
@IkaDemisto Nope, should we move the class to that file?

**Comment by IkaDemisto** [comment]
> @IkaDemisto Nope, should we move the class to that file?

Yes, I think so.

**Pull request merged**